### PR TITLE
[Feature] Update osf user model to store ssrn profile url [OSF-7313]

### PR DESCRIPTION
**Pull request by fabmiz**
## Purpose

Store user ssrn profile link in the database.

Currently, user ssrn profile link is not being stored to the database, hence does not
show social profile view and user search.

## Changes
- Update osf user model to store ssn profile link.

## Ticket
https://openscience.atlassian.net/browse/OSF-7313

### CODE REVIEW

**Comment by brianjgeiger** [comment]
I'm guessing we'll need to migrate_search for this.

File: osf_tests/test_search_views.py [hunk 1]
```diff
@@ -95,6 +95,80 @@ def test_search_projects(self):
...
+        assert_true(res.json['results'][1]['social'])
+        assert_not_equal(res.json['results'][0]['social']['ssrn'], res.json['results'][1]['social']['ssrn'])
+        assert_not_equal(res.json['results'][0]['social']['github'], res.json['results'][1]['social']['github'])
+
+        res = self.app.get(url, {'q': 'Uwase'})
+
+        assert_equal(res.status_code, 200)
+        assert_equal(len(res.json['results']), 1)
+        assert_true(res.json['results'][0]['social'])
+        assert_raises(KeyError, lambda: res.json['results'][0]['social']['ssrn'])
```
Comment (alexschiller) [inline osf_tests/test_search_views.py]: Probably better:
assert_not_in('ssrn', res.json['results'][0]['social'])

File: osf_tests/test_search_views.py [hunk 2]
```diff
@@ -95,6 +95,80 @@ def test_search_projects(self):
         res = self.app.get(url, {'q': self.project.title})
         assert_equal(res.status_code, 200)
 
+    def test_search_user(self):
+
+        url = '/api/v1/search/user/'
```
Comment (alexschiller) [inline osf_tests/test_search_views.py]: Didn't  `url = api_url_for('search_contributor')` return this url?

Comment (brianjgeiger) [inline osf_tests/test_search_views.py]: Also, now that I think about it, I think this is not meant for the contributor search but for global search.

Comment (fabmiz) [inline osf_tests/test_search_views.py]: @alexschiller `api_url_for('search_contributor')` returns `/api/v1/user/search/` not `/api/v1/search/user/`. The corresponding requests return different results.

Comment (alexschiller) [inline osf_tests/test_search_views.py]: @brianjgeiger Was the intent for these changes to be for contributor search and not global? If so @fabmiz may need to change this around more. I don't know enough about the v1 search routes to comment.

Comment (brianjgeiger) [inline osf_tests/test_search_views.py]: If we check the screenshots on the original ticket, they show what look to be global search page results.

Comment (fabmiz) [inline osf_tests/test_search_views.py]: I believe I am testing global user search with `/api/v1/search/user/`

Comment (brianjgeiger) [inline osf_tests/test_search_views.py]: Verified. That's fine, then.

**Pull request merged**